gcloud/logging/client.py

```diff
@@ -47,6 +49,28 @@ class Client(JSONClient):
     """
 
     _connection_class = Connection
+    _logging_api = _sinks_api = _metrics_api = None
+
+    @property
+    def logging_api(self):
+        """Helper for logging-related API calls."""
+        if self._logging_api is None:
+            self._logging_api = _LoggingAPI(self.connection)
```

Comment:
It is the same pattern as in `gcloud.pubsub.client`.  We could move the initialization into `__init__()`, but it felt simpler to me to do it on demand.